contracts/lease/src/api/open/mod.rs

```diff
@@ -108,6 +110,28 @@ impl InterestPaymentSpec {
     }
 }
 
+#[derive(Serialize, Deserialize, Clone)]
+#[cfg_attr(test, derive(Debug))]
+#[serde(deny_unknown_fields, rename = "loan", rename_all = "snake_case")]
+pub struct PositionSpec {
+    pub liability: Liability,
+    min_asset: LpnCoin,
+    min_sell_asset: LpnCoin,
+}
+
+impl PositionSpec {
+    pub fn new(liability: Liability, min_asset: LpnCoin, min_sell_asset: LpnCoin) -> Self {
```

Comment:
this might be needed only for testing ... so please put it behind a cfg

Reply:
Actually, it is used in the try_into_dto method of contracts/lease/src/lease/mod.dto where Position::new()  is called

Reply:
it should not be the case since the `PositionDTO` should not create a `PositionSpec` each time it's created. The spec is immutable and only ser/de in production